### PR TITLE
build: updating utils dependency

### PR DESCRIPTION
**Pull request by aorumbayev**
# Proposed changes

- Updating to latest algokit utils version

## Further notes

- Beaker would need to be changed to prod release once token is fixed and its published

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -19,7 +19,7 @@ pytest-cov = "*"
 pip-audit = "^2.5.5"
 python-semantic-release = "^7.33.4"
 pre-commit = "^3.3.2"
-beaker-pyteal = "^1.0.0"
+beaker-pyteal = "1.1.0b1"
```
Comment (robdmoore) [inline pyproject.toml:22]: Is this right?

File: examples/helloworld/client.py [hunk 2]
```diff
@@ -134,8 +134,8 @@ def _as_dict(data: typing.Any, *, convert_all: bool = True) -> dict[str, typing.
 
 def _convert_transaction_parameters(
     transaction_parameters: algokit_utils.TransactionParameters | None,
-) -> algokit_utils.CommonCallParametersDict:
-    return typing.cast(algokit_utils.CommonCallParametersDict, _as_dict(transaction_parameters))
+) -> algokit_utils.TransactionParameters:
```
Comment (robdmoore) [inline examples/helloworld/client.py:137]: What's the difference between these types?

**Review by robdmoore — APPROVED** [review]

File: examples/helloworld/client.py [hunk 3]
```diff
@@ -134,8 +134,8 @@ def _as_dict(data: typing.Any, *, convert_all: bool = True) -> dict[str, typing.
 
 def _convert_transaction_parameters(
     transaction_parameters: algokit_utils.TransactionParameters | None,
-) -> algokit_utils.CommonCallParametersDict:
```
Comment (aorumbayev) [inline examples/helloworld/client.py:137]: @daniel-makerx just wanted to confirm if this is a typo on utils or whether i should bump utils and export the TransactionParametersDict object explicitly. 

The deprecation note on  CommonCallParametersDict mentions `"""Deprecated, use TransactionParametersDict instead""" ` but that object isn't exported in init.py on py utils, however a dataclass named TransactionParameters is exported and seems to be working in this context as well. But i am not sure if we meant to say `use TransactionParameters  instead` in the deprecation warning or whether we forgot to export the `TransactionParametersDict` in init.py. 

https://github.com/algorandfoundation/algokit-utils-py/pull/46 - thats the pr that introduced the change. Let me know if you could clarify in a reply and ill apply appropriate changes on the py utils repo

File: pyproject.toml (hunk 1 above)
Comment (aorumbayev) [reply pyproject.toml:22]: @robdmoore see `further notes` - will change this as soon as devrels publish the beaker 1.1.0

File: examples/helloworld/client.py (hunk 2 above)
Comment (aorumbayev) [reply examples/helloworld/client.py:137]: @robdmoore left a comment for Dan to clarify

File: examples/helloworld/client.py (hunk 3 above)
Comment (daniel-makerx) [reply examples/helloworld/client.py:137]: So the idea was you could use explicit types like the `TransactionParameters` dataclass e.g. `TransactionParameters(sender=some_sender)`, or alternatively a dict with typed keys/values e.g. `{"sender": some_sender}`. But yeah looks like forgot to add it to init.py in algokit-utils, as this function is supposed to be returning a dict, will need to add it.

**Pull request merged**